r23740 | karl | 2011-08-28 21:14:50 -0300 (Sun, 28 Aug 2011) | 1 line

branch2011: import off-by-one fix for enctex/xetex initialization (original r23187, see also email http://tug.org/pipermail/tldistro/2011q3/000120.html)
Index: Build/source/texk/web2c/xetexdir/xetex.ch
===================================================================
--- Build/source/texk/web2c/xetexdir/xetex.ch	(revision 23739)
+++ Build/source/texk/web2c/xetexdir/xetex.ch	(revision 23740)
@@ -328,7 +328,7 @@
 {Initialize enc\TeX\ data.}
 for i:=0 to 255 do mubyte_read[i]:=null;
 for i:=0 to 255 do mubyte_write[i]:=0;
-for i:=0 to 128 do mubyte_cswrite[i]:=null;
+for i:=0 to 127 do mubyte_cswrite[i]:=null;
 mubyte_keep := 0; mubyte_start := false;
 write_noexpanding := false; cs_converting := false;
 special_printing := false; message_printing := false;
Index: Build/source/texk/web2c/enctex.ch
===================================================================
--- Build/source/texk/web2c/enctex.ch	(revision 23739)
+++ Build/source/texk/web2c/enctex.ch	(revision 23740)
@@ -42,7 +42,7 @@
 {Initialize enc\TeX\ data.}
 for i:=0 to 255 do mubyte_read[i]:=null;
 for i:=0 to 255 do mubyte_write[i]:=0;
-for i:=0 to 128 do mubyte_cswrite[i]:=null;
+for i:=0 to 127 do mubyte_cswrite[i]:=null;
 mubyte_keep := 0; mubyte_start := false;
 write_noexpanding := false; cs_converting := false;
 special_printing := false; message_printing := false;
